lib/presentation/model/search/presentation_server_side_empty_search.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:fluffychat/presentation/model/search/presentation_server_side.dart';
+
+class PresentationServerSideEmptySearch extends PresentationServerSide {
```

Comment:
you can create `PresentationServerSideEmptySearch`  extends the `PresentationServerSideSearch` with searchResults = []